### PR TITLE
Update Polish (pl) localization of Mozilla.org 160726

### PR DESCRIPTION
**Pull request by piotrdrag**
needs-review: @splewako

### CODE REVIEW

File: pl/firefox/australis/firefox_tour.lang [hunk 1]
```diff
@@ -26,7 +26,7 @@ Dodawaj i przeglądaj zakładki w jednym miejscu
 
 
 ;Learn more about the new Firefox
-Dowiedz się więcej o nowym Firefoksie
+Więcej informacji o nowym Firefoksie
```
Comment (splewako) [inline pl/firefox/australis/firefox_tour.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/firefox/desktop/customize.lang [hunk 2]
```diff
@@ -68,7 +68,7 @@ Wypróbuj teraz
 
 
 ;Learn more
-Dowiedz się więcej
+Więcej informacji
```
Comment (splewako) [inline pl/firefox/desktop/customize.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/firefox/desktop/customize.lang [hunk 3]
```diff
@@ -116,7 +116,7 @@ Dostęp do Twoich zakładek, historii, haseł i innych danych sprawia, że Fire
 
 
 ;Learn more about Sync
-Dowiedz się więcej o synchronizacji
+Więcej informacji o synchronizacji
```
Comment (splewako) [inline pl/firefox/desktop/customize.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/firefox/desktop/trust.lang [hunk 4]
```diff
@@ -167,7 +167,7 @@ Ograniczanie śledzenia to innowacja Firefoksa, która umożliwia wskazanie, jak
 
 # Obsolete string, will be removed after the Firefox 42 campaign
 ;Learn more about Do Not Track
-Dowiedz się więcej o ograniczaniu śledzenia
+Więcej informacji o ograniczaniu śledzenia
```
Comment (splewako) [inline pl/firefox/desktop/trust.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/firefox/includes/mwc_2015_schedule.lang [hunk 5]
```diff
@@ -7,62 +7,62 @@ Poniedziałek
...
 
 
 ;Building Mozilla Values into Firefox OS
-Building Mozilla Values into Firefox OS
+Wdrażanie wartości Mozilli w Firefox OS
 
 
 ;Firefox OS. Beyond Smartphones.
-Firefox OS. Beyond Smartphones.
+Firefox OS. Nie tylko smartfony.
```
Comment (splewako) [inline pl/firefox/includes/mwc_2015_schedule.lang]: Please try to not to use "smartfony", simple "telefony" should work good here.

File: pl/firefox/includes/mwc_2015_schedule.lang [hunk 6]
```diff
@@ -7,62 +7,62 @@ Poniedziałek
...
 
 
 ;Firefox Marketplace. Why We Chose the Web.
-Firefox Marketplace. Why We Chose the Web.
+Firefox Marketplace. Dlaczego wybraliśmy sieć.
 
 
 ;Firefox OS, the Web of Things and Open Web Hardware
-Firefox OS, the Web of Things and Open Web Hardware
+Firefox OS, Internet Rzeczy i sprzęt otwartej sieci
```
Comment (splewako) [inline pl/firefox/includes/mwc_2015_schedule.lang]: "Internet rzeczy"

File: pl/firefox/new/horizon.lang [hunk 7]
```diff
@@ -55,7 +55,7 @@ Pobierz Firefoksa na Androida i iOSa.
 
 
 ;Learn more about Firefox for Mobile
-Dowiedz się więcej o Firefoksie Mobile
+Więcej informacji o Firefoksie Mobile
```
Comment (splewako) [inline pl/firefox/new/horizon.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/firefox/os/index-new.lang [hunk 8]
```diff
@@ -231,7 +231,7 @@ Mozilla jest organizacją non-profit, wspieraną przez zaangażowaną globalną
 
 
 ;Learn more about Mozilla
-Dowiedz się więcej o Mozilli
+Więcej informacji o Mozilli
```
Comment (splewako) [inline pl/firefox/os/index-new.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/firefox/os/tv.lang [hunk 9]
```diff
@@ -18,7 +18,7 @@ Firefox OS w TV
 
 
 ;We’re bringing the power of the Web to Panasonic televisions again in 2016.
-We’re bringing the power of the Web to Panasonic televisions again in 2016.
+W 2016 roku ponownie dostarczymy potęgę sieci do telewizorów Panasonic.
```
Comment (splewako) [inline pl/firefox/os/tv.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/firefox/private-browsing.lang [hunk 10]
```diff
@@ -104,7 +104,7 @@ Ochronę można łatwo wyłączyć dla pojedynczej strony lub całkowicie. To za
 
 
 ;Learn more about the Control Center
-Dowiedz się więcej
+Więcej informacji
```
Comment (splewako) [inline pl/firefox/private-browsing.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/foundation/annualreport/2012/index.lang [hunk 11]
```diff
@@ -95,7 +95,7 @@ Wygenerowanie większości naszych przychodów było możliwe dzięki wbudowanej
 
 
 ;A growing percentage of our revenue also comes from support from the public, including grants and individual donors like you. These donations are directly applied to important projects such as Webmaker, Open Badges and Mozilla-Knight Open News that extend Mozilla's influence in new areas.
-Rosnącym odsetkiem naszych przychodów staje się wsparcie społeczeństwa poprzez dotacje od firm i osób prywatnych, takich jak Ty. Przychody z tego tytułu są kierowane na ważne działania rozszerzające wpływ Mozilli w zupełnie nowych dziedzinach. Wśród takich projektów są między innymi: Webmaker, Open Badges i Mozilla-Knight Open News.
+Rosnącym odsetkiem naszych przychodów staje się wsparcie społeczeństwa poprzez darowizny od firm i osób prywatnych, takich jak Ty. Przychody z tego tytułu są kierowane na ważne działania rozszerzające wpływ Mozilli w zupełnie nowych dziedzinach. Wśród takich projektów są między innymi: Webmaker, Open Badges i Mozilla-Knight Open News.
```
Comment (splewako) [inline pl/foundation/annualreport/2012/index.lang]: I don't think this is correct, see also http://bugs.aviary.pl/show_bug.cgi?id=4981

File: pl/main.lang [hunk 12]
```diff
@@ -141,7 +141,7 @@ Firefox: nie dla zysku, niekorporacyjny, nieuległy. Wybierając Firefoksa wybie
 
 
 ;Donate
-Dotacje
+Darowizny
```
Comment (splewako) [inline pl/main.lang]: I don't think this is correct, see also http://bugs.aviary.pl/show_bug.cgi?id=4981

File: pl/mozorg/about/manifesto.lang [hunk 13]
```diff
@@ -34,7 +34,7 @@ Zobacz więcej…
 
 
 ;Learn more
-Dowiedz się więcej
+Więcej informacji
```
Comment (splewako) [inline pl/mozorg/about/manifesto.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/mozorg/about/manifesto.lang [hunk 14]
```diff
@@ -305,11 +305,11 @@ Zatweetuj
 
 
 ;Learn more about…
-Dowiedz się więcej…
+Więcej informacji…
```
Comment (splewako) [inline pl/mozorg/about/manifesto.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/mozorg/about/manifesto.lang [hunk 15]
```diff
@@ -305,11 +305,11 @@ Zatweetuj
...
 
 
 ;Donating to Mozilla
-Donacje na rzecz Mozilli
+Darowizny na rzecz Mozilli
```
Comment (splewako) [inline pl/mozorg/about/manifesto.lang]: I don't think this is correct, see also http://bugs.aviary.pl/show_bug.cgi?id=4981

File: pl/mozorg/home/index.lang [hunk 16]
```diff
@@ -45,7 +45,7 @@ Wypróbuj przeglądanie prywatne
 
 # Small tile. Link points to https://advocacy.mozilla.org
 ;Learn about Mozilla Advocacy
-Dowiedz się więcej o Mozilla Advocacy
+Więcej informacji o Mozilla Advocacy
```
Comment (splewako) [inline pl/mozorg/home/index.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/mozorg/plugincheck-redesign.lang [hunk 17]
```diff
@@ -17,39 +17,39 @@ Sprawdź swoje wtyczki
 
 
 ;Plugins power the videos, animations and games in your Firefox. Keep an eye on their status and keep them up to date.
-Plugins power the videos, animations and games in your Firefox. Keep an eye on their status and keep them up to date.
+Wtyczki umożliwiają oglądanie filmów i animacji oraz granie w gry w Firefoksie. Pilnuj, aby były aktualne.
 
 
 ;Be sure to restart your Firefox after making all recommended updates.
-Be sure to restart your Firefox after making all recommended updates.
+Uruchom ponownie Firefoksa dopiero po zainstalowanie wszystkich zalecanych uaktualnień.
```
Comment (splewako) [inline pl/mozorg/plugincheck-redesign.lang]: zainstalowaniu

File: pl/mozorg/plugincheck-redesign.lang [hunk 18]
```diff
@@ -81,67 +81,67 @@ Porady, wskazówki i wiadomości ze świata Firefoksa
...
+Pomoc Mozilli
 
 
 ;What is a plugin?
 Czym jest wtyczka?
 
 
 ;Plugins are bits of software that power videos, animation and games in your browser.
-Plugins are bits of software that power videos, animation and games in your browser.
+Wtyczki to małe programy umożliwiające oglądanie filmów i animacji oraz granie w gry w przeglądarce.
```
Comment (splewako) [inline pl/mozorg/plugincheck-redesign.lang]: Please remove "małe"

File: pl/tabzilla/tabzilla.lang [hunk 19]
```diff
@@ -93,7 +93,7 @@ Wybiegaj myślą
 
 # Link for slogan
 ;Learn all about Firefox OS
-Dowiedz się więcej o Firefox OS
+Więcej informacji o Firefox OS
```
Comment (splewako) [inline pl/tabzilla/tabzilla.lang]: This doesn't look good. Please ensure that the file is UTF-8 encoded.

File: pl/thunderbird/start/release.lang [hunk 20]
```diff
@@ -7,43 +7,43 @@ Witaj w <span>Thunderbirdzie</span>
...
 
 
 ;Thunderbird is the leading open source, cross-platform email and calendaring client, free for business and personal use. We want it to stay secure and become even better.
-Thunderbird is the leading open source, cross-platform email and calendaring client, free for business and personal use. We want it to stay secure and become even better.
+Thunderbird to wiodący międzyplatformowy klient poczty i kalendarza open source, darmowy dla przedsiębiorstw i użytku osobistego. Chcemy, aby nadal był bezpieczny i stał się jeszcze lepszy.
 
 
 ;If you like Thunderbird, please consider a donation!
-If you like Thunderbird, please consider a donation!
+Jeśli lubisz Thunderbirda, rozważ wpłacenie darowizny!
```
Comment (splewako) [inline pl/thunderbird/start/release.lang]: I don't think this is correct ("darowizny"), see also http://bugs.aviary.pl/show_bug.cgi?id=4981

File: pl/thunderbird/start/release.lang [hunk 21]
```diff
@@ -7,43 +7,43 @@ Witaj w <span>Thunderbirdzie</span>
...
 
 
 ;By donating, you will help us to continue delivering an ad-free top-notch email client.
-By donating, you will help us to continue delivering an ad-free top-notch email client.
+Wpłacając, pomożesz nam nadal dostarczać najlepszego klienta poczty, całkowicie wolnego od reklam.
 
 
 ;Make a donation
-Make a donation
+Wpłać darowiznę
```
Comment (splewako) [inline pl/thunderbird/start/release.lang]: I don't think this is correct ("darowizny"), see also http://bugs.aviary.pl/show_bug.cgi?id=4981

File: pl/thunderbird/start/release.lang [hunk 22]
```diff
@@ -7,43 +7,43 @@ Witaj w <span>Thunderbirdzie</span>
...
 
 
 ;Other ways to contribute to Thunderbird
-Other ways to contribute to Thunderbird
+Inne sposoby wspomożenia Thunderbirda
```
Comment (splewako) [inline pl/thunderbird/start/release.lang]: wspomagania?

File: pl/firefox/australis/firefox_tour.lang (hunk 1 above)
Comment (flodolo) [inline pl/firefox/australis/firefox_tour.lang]: I'm not sure if something broke with GitHub recently, I've seen it in another repository.
https://github.com/piotrdrag/www.mozilla.org/blob/ea0bdee4f0978e755db3edb9b8f2ae3a20ff34f5/pl/firefox/australis/firefox_tour.lang#L28

File: pl/firefox/includes/mwc_2015_schedule.lang (hunk 5 above)
Comment (piotrdrag) [inline pl/firefox/includes/mwc_2015_schedule.lang]: Fixed.

File: pl/firefox/includes/mwc_2015_schedule.lang (hunk 6 above)
Comment (piotrdrag) [inline pl/firefox/includes/mwc_2015_schedule.lang]: Fixed.

File: pl/mozorg/plugincheck-redesign.lang (hunk 17 above)
Comment (piotrdrag) [inline pl/mozorg/plugincheck-redesign.lang]: Fixed.

File: pl/mozorg/plugincheck-redesign.lang (hunk 18 above)
Comment (piotrdrag) [inline pl/mozorg/plugincheck-redesign.lang]: Done.

File: pl/thunderbird/start/release.lang (hunk 22 above)
Comment (piotrdrag) [inline pl/thunderbird/start/release.lang]: Fixed.

File: pl/thunderbird/start/release.lang (hunk 21 above)
Comment (piotrdrag) [inline pl/thunderbird/start/release.lang]: Changed.

File: pl/thunderbird/start/release.lang (hunk 20 above)
Comment (piotrdrag) [inline pl/thunderbird/start/release.lang]: Changed.

File: pl/foundation/annualreport/2012/index.lang (hunk 11 above)
Comment (piotrdrag) [inline pl/foundation/annualreport/2012/index.lang]: Change dropped.

File: pl/main.lang (hunk 12 above)
Comment (piotrdrag) [inline pl/main.lang]: Change dropped.

File: pl/mozorg/about/manifesto.lang (hunk 15 above)
Comment (piotrdrag) [inline pl/mozorg/about/manifesto.lang]: Changed to "Dotacje" for consistency, is it better?

File: pl/firefox/australis/firefox_tour.lang (hunk 1 above)
Comment (piotrdrag) [inline pl/firefox/australis/firefox_tour.lang]: It's UTF-8 on my end.

Comment (splewako) [inline pl/firefox/australis/firefox_tour.lang]: OK, let's double check after integrating.

File: pl/mozorg/about/manifesto.lang (hunk 15 above)
Comment (splewako) [inline pl/mozorg/about/manifesto.lang]: Not perfect but better.

**Pull request merged**